pack/parallel/rod.json

```diff
@@ -0,0 +1,69 @@
+[
+	{
+		"back_flavor": "As a respected librarian at Miskatonic University, Daisy had always felt that books were the most important thing in her life. She explored in fiction what she abhorred in life: horror, violence, fear. Then, she stumbled across the John Dee translation of the Necronomicon. It was blasphemous, unholy, and too awful to be read. But given her studies in obscure and occult subjects, Daisy knew there was more truth than fiction within the book's pages. She began to wonder what other secrets the restricted collection of the Orne Library held...",
+		"back_text": "<b>Deck size</b>: 30.\n<b>Deckbuilding Options</b>: [[Tome]] cards level 0-5, Seeker cards ([seeker]) level 0-3, Neutral cards level 0-5, up to five other level 0 Guardian and/or Mystic cards ([guardian] and/or [mystic]).\n<b>Deckbuilding Requirements</b> (do not count toward deck size): Daisy’s Tote Bag, The Necronomicon (John Dee Translation), 1 random basic weakness.",
+		"code": "90001",
+		"deck_limit": 1,
+		"deck_options": [
+			{"faction":["neutral"], "level":{"min":0, "max":5} },
+			{"faction":["seeker"], "level":{"min":0, "max":5} },
+			{"faction":["guardian", "mystic"], "level":{"min":0, "max":0}, "limit": 5, "error": "You cannot have more than 5 Guardian and/or Mystic cards" }
+		],
+		"deck_requirements": "size:30, card:01008:90002, card:01009:90003, random:subtype:basicweakness",
+		"double_sided": true,
+		"faction_code": "seeker",
+		"flavor": "\"I can shut the book, but I can never unread its words.\"",
+		"health": 5,
+		"illustrator": "Magali Villeneuve",
+		"is_unique": true,
+		"name": "Daisy Walker",
+		"pack_code": "rod",
+		"position": 1,
+		"quantity": 1,
+		"sanity": 7,
+		"skill_agility": 2,
+		"skill_intellect": 5,
+		"skill_combat": 2,
+		"skill_willpower": 1,
+		"subname": "The Librarian",
+		"text": "You get +1 [willpower] and +1 sanity for each [[Tome]] asset you control.\n[fast]: One at a time, resolve an [action] ability on each [[Tome]] asset you control, ignoring their [action] cost. (Limit once per game.)\n[elder_sign] effect: +1. You may return a [[Tome]] asset from your discard pile to your hand.",
+		"traits": "Miskatonic.",
+		"type_code": "investigator"
+	},
+	{
+		"code": "90002",
+		"cost": 2,
+		"deck_limit": 1,
+		"faction_code": "neutral",
+		"illustrator": "Dani Hartel",
+		"is_unique": true,
+		"name": "Daisy's Tote Bag",
+		"pack_code": "rod",
+		"position": 2,
+		"quantity": 1,
+		"restrictions": "investigator:90001",
```

Comment:
Does this restriction need to include 01002 (original Daisy)?

Reply:
So the existing promos only reference the 'original' card. @Kamalisk what do you think we should do for these Advanced ones, same thing?